lib/chat_web/controllers/zip_controller.ex

```diff
@@ -16,6 +16,7 @@ defmodule ChatWeb.ZipController do
   alias Chat.Messages.ExportHelper
   alias Chat.Rooms
   alias Chat.User
+  alias Chat.Utils
```

Comment:
`alias Chat.Utils, only: [short_hash: 1]`

and change code bellow respectively

Reply:
and point is to create functional component once. have this dependency in it. and reuse component in the system

Reply:
> `alias Chat.Utils, only: [short_hash: 1]`
> 
> and change code bellow respectively

the `:only` option is unsupported to alias